eventually-v0-5-0/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "eventually-v0-5-0"
+version = "0.5.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0.45"
+async-trait = "0.1.51"
+futures = "0.3.17"
```

Comment:
do you need `futures`? or does `futures_util` have everything you need in it? (it's smaller)